rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -990,6 +1000,11 @@ rmw_create_subscription(
   free_topic_name.cancel();
   free_rmw_subscription.cancel();
 
+  // rmw does not require GIDs for subscriptions, and GIDs in rmw_zenoh are not based on any ID of
+  // the underlying zenoh objects, so there is no need to collect a GID here
+  rmw_gid_t gid{};
```

Comment:
I will note that if you have tracing disabled (like I do locally), then this is a compile warning since `gid` goes unused.  Maybe at least put a `static_cast<void>(gid)` below this?

Reply:
added!